core/src/main/java/io/kestra/core/test/TestSuiteUid.java

```diff
@@ -0,0 +1,10 @@
+package io.kestra.core.test;
+
+import io.kestra.core.utils.IdUtils;
+
+public record TestSuiteUid(String tenant, String namespace, String id) {
```

Comment:
This seems overkill, you could directly use the IdUtils from the uuid() method.
KISS